Topics/Tech_Stacks/iOS.md

```diff
@@ -0,0 +1,56 @@
+# Creating Your First iOS App
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [Prerequistites](#)
+3. [Steps to Creating an iOS App](#steps-to-creating-an-ios-app)
+4. [Conclusion](#conclusion)
+5. [Additional Resources](#additional-resources)
+
+## Introduction
+<!-- intro -->
+With millions of users worldwide, Apple's iOS platform offers a vast audience for your apps, whether you aim to develop utility apps, games, or educational tools. Creating an iOS app not only unlocks the potental to reach a global audience but also provides an opportunity to learn new technologies and design principles. 
+
+## Prerequisites
+<!-- anything the user may need -->
+Before we begin, ensure you have the following:
+
+1. **Apple computer** with latest macOS version.
+2. **Xcode**. Install [here](https://apps.apple.com/us/app/xcode/id497799835?mt=12)!
```

Comment:
I think it would be useful to elaborate more on what these technologies would be used for/ why they are used

Reply:
Thoughts on this? 

```suggestion
2. **Xcode**: Apple's integrated development environment (IDE), available for free from the [Mac App Store](https://apps.apple.com/us/app/xcode/id497799835?mt=12). Used to develop programs in macOS, iOS, iPadOS, watchOS, tvOS, and visionOS.
```